Taskfile.yml

```
@@ -0,0 +1,145 @@
+    dir: "{{ .ROOT_DIR }} "
+    desc: Run tests
+    cmds:
+      - doppler run --  python3 run/manage.py test applications.API
+    requires:
+      vars:
+        - DOPPLER_TOKEN
+        - PATH_TO_DB_ROOT_CERT
+
+  run:
+    desc: Starts a Gunicorn Based Development Server that Reloads on Changes
+    dir: "{{ .ROOT_DIR}}" 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)